src/gpu/symmetry.jl

```diff
@@ -0,0 +1,46 @@
+using GPUArraysCore
+
+function accumulate_over_symmetries!(ρaccu::AbstractArray, ρin::AbstractArray,
+    basis::PlaneWaveBasis{T}, symmetries) where {T}
```

Comment:
Formatting

Reply:
Also, how much worse is this implementation compared to the CPU version ? Can we not just make this the CPU version, too ? It looks like it should not be very much worse than what we have.

Reply:
I agree. Avoiding code duplication is more important here. Also it's not all too bad. Just keep the appropriate comments to make sure it's clear that the GPU is the reason why things are done in a certain way.